cmake/krisp.cmake

```diff
@@ -6,9 +6,112 @@ if(NOT DEFINED ENV{KRISP_LIB})
+endif()
+
+find_library(LIBXNNPACK_ABSPATH NAMES XNNPACK PATHS $ENV{LIBRESAMPLE_LIB} NO_DEFAULT_PATH)
+if (NOT LIBXNNPACK_ABSPATH) 
+	message(FATAL_ERROR "Can't find libXNNPACK static library")
+endif()
+
+find_library(LIBZIP_ABSPATH NAMES zip PATHS $ENV{LIBRESAMPLE_LIB} NO_DEFAULT_PATH)
+if (NOT LIBZIP_ABSPATH) 
+	message(FATAL_ERROR "Can't find libzip static library")
+endif()
+
+find_library(LIBZ_ABSPATH NAMES z PATHS $ENV{LIBRESAMPLE_LIB} NO_DEFAULT_PATH)
+if (NOT LIBZ_ABSPATH) 
+	message(FATAL_ERROR "Can't find libz static library")
+endif()
+
+find_library(LIBZSTD_ABSPATH NAMES zstd PATHS $ENV{LIBRESAMPLE_LIB} NO_DEFAULT_PATH)
+if (NOT LIBZSTD_ABSPATH) 
+	message(FATAL_ERROR "Can't find libzsd static library")
+endif()
+
+find_library(LIBBZ2_ABSPATH NAMES bz2 PATHS $ENV{LIBRESAMPLE_LIB} NO_DEFAULT_PATH)
+if (NOT LIBBZ2_ABSPATH) 
+	message(FATAL_ERROR "Can't find libbz2 static library")
+endif()
+
+find_library(LIBCPUINFO_ABSPATH NAMES cpuinfo PATHS $ENV{LIBRESAMPLE_LIB} NO_DEFAULT_PATH)
+if (NOT LIBCPUINFO_ABSPATH) 
+	message(FATAL_ERROR "Can't find libcpuinfo static library")
+endif()
+
+find_library(LIBLZMA_ABSPATH NAMES lzma PATHS $ENV{LIBRESAMPLE_LIB} NO_DEFAULT_PATH)
+if (NOT LIBLZMA_ABSPATH) 
+	message(FATAL_ERROR "Can't find liblzma static library")
+endif()
+
+find_library(LIBPTHREADPOOL_ABSPATH NAMES pthreadpool PATHS $ENV{LIBRESAMPLE_LIB} NO_DEFAULT_PATH)
+if (NOT LIBPTHREADPOOL_ABSPATH) 
+	message(FATAL_ERROR "Can't find pthreadpool static library")
+endif()
+
+find_library(LIBFFTW3F_ABSPATH NAMES fftw3f PATHS $ENV{LIBRESAMPLE_LIB} NO_DEFAULT_PATH)
+if (NOT LIBFFTW3F_ABSPATH) 
+	message(FATAL_ERROR "Can't find libfftw3f static library")
+endif()
+
+find_library(LIBFFTW3F_THREADS_ABSPATH NAMES fftw3f_threads PATHS $ENV{LIBRESAMPLE_LIB} NO_DEFAULT_PATH)
+if (NOT LIBFFTW3F_THREADS_ABSPATH) 
+	message(FATAL_ERROR "Can't find libfftw3f_threads static library")
+endif()
+
+find_library(LIBONNXRUNTIME_ABSPATH NAMES onnxruntime PATHS $ENV{LIBRESAMPLE_LIB} NO_DEFAULT_PATH)
+if (NOT LIBONNXRUNTIME_ABSPATH) 
+	message(FATAL_ERROR "Can't find libonnxruntime static library")
+endif()
+
+set(KRISP_LIBS 
+	${LIBKRISP_ABSPATH}
+	${LIBRESAMPLE_ABSPATH}
+	${LIBXNNPACK_ABSPATH}
+	${LIBZIP_ABSPATH}
+	${LIBZ_ABSPATH}
+	${LIBZSTD_ABSPATH}
+	${LIBBZ2_ABSPATH}
+	${LIBCPUINFO_ABSPATH}
+	${LIBLZMA_ABSPATH}
+	${LIBPTHREADPOOL_ABSPATH}
+	${LIBFFTW3F_ABSPATH}
+	${LIBFFTW3F_THREADS_ABSPATH}
+	${LIBONNXRUNTIME_ABSPATH}
+	/Users/atatalyan/dev/krisp-sdk/krisp-audio-sdk-8.0.0/external/libctc-decoder.a
+	/Users/atatalyan/dev/krisp-sdk/krisp-audio-sdk-8.0.0/external/libsentencepiece.a
+	/Users/atatalyan/dev/krisp-sdk/krisp-audio-sdk-8.0.0/external/libcppnini.a
+	/Users/atatalyan/dev/krisp-sdk/krisp-audio-sdk-8.0.0/external/libopenfst-useful.a
+	/Users/atatalyan/dev/krisp-sdk/krisp-audio-sdk-8.0.0/external/libpynini-useful.a
+	/Users/atatalyan/dev/gitrepos/krisp-media-sdk/build/krisp-nc-processor/lib/libkrisp-nc-processor.a
```

Comment:
let's look into the single library issue causing the linker errors in Mac/Linux.